taplo.toml

```diff
@@ -0,0 +1,26 @@
+[formatting]
+# Align consecutive entries vertically.
+align_entries = false
+# Append trailing commas for multi-line arrays.
+array_trailing_comma = true
+# Expand arrays to multiple lines that exceed the maximum column width.
+array_auto_expand = true
+# Collapse arrays that don't exceed the maximum column width and don't contain comments.
+array_auto_collapse = false
+# Omit white space padding from single-line arrays
+compact_arrays = true
+# Omit white space padding from the start and end of inline tables.
+compact_inline_tables = false
+# Maximum column width in characters, affects array expagit nsion and collapse, this doesn't take whitespace into account.
+# Note that this is not set in stone, and works on a best-effort basis.
+column_width = 160
```

Comment:
If you disagree with the above suggestion of setting this to 100 instead of 160, let's still update the comment (there is a minor typo there as well, "expagit nsion" -> "expansion". So if you do not merge any of my suggestions on this, be sure to fix that).

```suggestion
# Maximum column width in characters.
column_width = 160
```